src/main/java/net/minecraftforge/event/world/WorldEvent.java

```diff
@@ -118,4 +124,44 @@ public ServerLevelData getSettings()
             return settings;
         }
     }
+
+    /**
+     * Called by NaturalSpawner to gather a list of all possible entities that can spawn at the specified location.
+     * If an entry is added to the list, it needs to be a globally unique instance.
+     * The event is called in {@link NaturalSpawner#getRandomSpawnMobAt()}.
```

Comment:
This should link to the FQN so that the IDE can properly link them when clicking to go to target even if it shows an error due to it being private. `{@link net.minecraft.world.level.NaturalSpawner#getRandomSpawnMobAt(ServerLevel, StructureManager, ChunkGenerator, MobCategory, RandomSource, BlockPos)}`